src/arch/armv8/armv8-a/inc/arch/smmuv2.h

```diff
@@ -8,108 +8,108 @@
-#define SMMUV2_CR0_GCFGFIE  (0x1 << 5)
-#define SMMUV2_CR0_USFCFG   (0x1 << 10)
-#define SMMUV2_CR0_SMCFCFG  (0x1 << 21)
-#define SMMUV2_CR0_CLIENTPD (0x1 << 0)
-
-#define SMMUV2_CR0_CLEAR(cr0)     (cr0 & (0x3 << 30 | 0x1 << 11))
-
-#define SMMUV2_IDR0_MASK         (0xFF)
-#define SMMUV2_IDR0_S2TS_BIT     (0x1 << 30)
-#define SMMUV2_IDR0_SMS_BIT      (0x1 << 27)
-#define SMMUV2_IDR0_CTTW_BIT     (0x1 << 14)
-#define SMMUV2_IDR0_BTM_BIT      (0x1 << 13)
-
-#define SMMUV2_IDR1_PAGESIZE_BIT    (0x1 << 31)
-#define SMMUV2_IDR1_NUMCB_OFF       (0)
-#define SMMUV2_IDR1_NUMCB_LEN       (8)
-#define SMMUV2_IDR1_NUMPAGEDXB_OFF  (28)
-#define SMMUV2_IDR1_NUMPAGEDXB_LEN  (3)
-
-#define SMMUV2_IDR2_PTFSv8_4kB_BIT (0x1 << 12)
-#define SMMUV2_IDR2_OAS_OFF         (4)
-#define SMMUV2_IDR2_OAS_LEN         (4)
-#define SMMUV2_IDR2_IAS_OFF         (0)
-#define SMMUV2_IDR2_IAS_LEN         (4)
-
-#define SMMUV2_IDR7_MAJOR_OFF       (4)
-#define SMMUV2_IDR7_MAJOR_LEN       (4)
-
-#define SMMU_SMR_ID_OFF 0
-#define SMMU_SMR_ID_LEN 15
-#define SMMU_SMR_ID(smr) bit32_extract(smr, SMMU_SMR_ID_OFF, SMMU_SMR_ID_LEN)
-
-#define SMMU_ID_MSK BIT32_MASK(0, SMMU_SMR_ID_LEN)
-
-#define SMMU_SMR_MASK_OFF 16
-#define SMMU_SMR_MASK_LEN 15
-#define SMMU_SMR_MASK(smr) bit32_extract(smr, SMMU_SMR_MASK_OFF, SMMU_SMR_MASK_LEN)
-
-#define SMMUV2_SMR_VALID (0x1 << 31)
-
-#define S2CR_IMPL_OFF         (30)
-#define S2CR_IMPL_LEN         (2)
-#define S2CR_IMPL_MASK        BIT32_MASK(S2CR_IMPL_OFF, S2CR_IMPL_LEN)
-
-#define S2CR_TRANSIENTCFG_OFF         (28)
-#define S2CR_TRANSIENTCFG_LEN         (2)
-#define S2CR_TRANSIENTCFG_MASK        BIT32_MASK(S2CR_TRANSIENTCFG_OFF, S2CR_TRANSIENTCFG_LEN)
+#define SMMUV2_CR0_GFRE                 (0x1 << 1)
+#define SMMUV2_CR0_GFIE                 (0x1 << 2)
+#define SMMUV2_CR0_GCFGFRE              (0x1 << 4)
+#define SMMUV2_CR0_GCFGFIE              (0x1 << 5)
+#define SMMUV2_CR0_USFCFG               (0x1 << 10)
+#define SMMUV2_CR0_SMCFCFG              (0x1 << 21)
+#define SMMUV2_CR0_CLIENTPD             (0x1 << 0)
+
+#define SMMUV2_CR0_CLEAR(cr0)           (cr0 & (0x3 << 30 | 0x1 << 11))
+
+#define SMMUV2_IDR0_MASK                (0xFF)
+#define SMMUV2_IDR0_S2TS_BIT            (0x1 << 30)
+#define SMMUV2_IDR0_SMS_BIT             (0x1 << 27)
+#define SMMUV2_IDR0_CTTW_BIT            (0x1 << 14)
+#define SMMUV2_IDR0_BTM_BIT             (0x1 << 13)
+
+#define SMMUV2_IDR1_PAGESIZE_BIT        (0x1 << 31)
+#define SMMUV2_IDR1_NUMCB_OFF           (0)
+#define SMMUV2_IDR1_NUMCB_LEN           (8)
+#define SMMUV2_IDR1_NUMPAGEDXB_OFF      (28)
+#define SMMUV2_IDR1_NUMPAGEDXB_LEN      (3)
+
+#define SMMUV2_IDR2_PTFSv8_4kB_BIT      (0x1 << 12)
+#define SMMUV2_IDR2_OAS_OFF             (4)
+#define SMMUV2_IDR2_OAS_LEN             (4)
+#define SMMUV2_IDR2_IAS_OFF             (0)
+#define SMMUV2_IDR2_IAS_LEN             (4)
+
+#define SMMUV2_IDR7_MAJOR_OFF           (4)
+#define SMMUV2_IDR7_MAJOR_LEN           (4)
+
+#define SMMU_SMR_ID_OFF                 0
```

Comment:
is there any option that can force this macro defines to be enclosed in parenthesis? To maintain consistency